svc/WebSocketService.ts

```diff
@@ -193,15 +200,44 @@ export class WebSocketService extends HoistService {
     private heartbeatOrReconnect() {
         this.updateConnectedStatus();
         if (this.connected) {
-            this.sendMessage({topic: this.HEARTBEAT_TOPIC, data: 'ping'});
+            // We have a channel key, so we successfully registered with the server and have not
+            // received a disconnect message. We should be receiving at least heartbeat messages,
+            // but have observed cases where "something" interrupted connectivity in a surprising
+            // way and no new inbound messages were arriving, even with the socket reporting open
+            // and accepting outbound messages to send. Detect that case here.
```

Comment:
My hope is that we might learn more and could remove or edit this 🤷 comment, but for now I wanted to document/acknowledge the weirdness in-place here in the code, at least to some degree....